xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -484,12 +484,14 @@ bool CVideoPlayerAudio::ProcessDecoderOutput(DVDAudioFrame &audioframe)
       if (!m_audioSink.Create(audioframe, m_streaminfo.codec, m_synctype == SYNC_RESAMPLE))
         CLog::Log(LOGERROR, "%s - failed to create audio renderer", __FUNCTION__);
 
-      m_audioSink.SetDynamicRangeCompression((long)(m_processInfo.GetVideoSettings().m_VolumeAmplification * 100));
-
       if (m_syncState == IDVDStreamPlayer::SYNC_INSYNC)
         m_audioSink.Resume();
     }
-
+	
+    // Apply VolumeAmplification from settings on playback
+    m_audioSink.SetDynamicRangeCompression(
+	    static_cast<long>(m_processInfo.GetVideoSettings().m_VolumeAmplification * 100));
+	
```

Comment:
tabs on both lines

Reply:
I originally had spaces per the coding guidelines, so not sure why it added tabs, but should be fixed now.

Reply:
all good :) could be your IDE does it automatically. there still seems to be something funky in your latest commit, jenkins points out

Reply:
I did see that, it looks like only some of the target platform builds were successful, I'm not sure why some failed, but looking at the error logs it doesn't look related to my changes. Is there a way to get Jenkins to rerun the build tasks?

Reply:
yeah, don‘t worry about the build failures. i‘ll close and reopen this pr now so the build should rerun. please avoid to push changes to your branch while an active build is running. also, after the style changes are applied, the pr should be squashed down to a single commit… but first things first

Reply:
Got it, I've squashed the commits down into one, we should be all good now. :)